.github/workflows/pull-request-python.yaml

```diff
@@ -11,23 +11,23 @@ on:
 
 jobs:
 
-  # Build is a single job to lint and test
   build:
     name: Build
     runs-on: ubuntu-latest
+    env:
+      IMAGE: ${{ github.repository }}
```

Comment:
This won't work, `github.repository` is going to be `gramLabs/something` which contains an illegal character for use in a container name ("L").

Reply:
Yeah this was based on a cursory glance at the github context variables docs ... I can just set the image name statically since it's not being published in this workflow